setup.py

```diff
@@ -2,18 +2,21 @@
 import re
 from pathlib import Path
 from setuptools import setup, find_packages
+from typing import Any
 
 
 ROOT = os.path.dirname(__file__)
 VERSION_RE = re.compile(r"""__version__ = ['"]([0-9.]+)['"]""")
 
 
-def get_version():
+def get_version() -> Any:
```

Comment:
Since there is the assertion that `version_value is not None` , could the return type here be `str`?

Reply:
Yes yes, that's even more specific to the get version method.